### DISCUSSION TITLE
[Script request] Wazuh

### DISCUSSION THREAD

**Discussion by Deepshah0308** [community-scripts/ProxmoxVE#166] [Request script]
### Application Name

Wazuh

### Website

https://wazuh.com/

### Description

Wazuh delivers robust security monitoring and protection for your IT assets using its Security Information and Event Management (SIEM) and Extended Detection and Response (XDR) capabilities. Wazuh use cases are designed to safeguard your digital assets and enhance your organization's cybersecurity posture.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by MickLesk** [comment]
```log
INFO: The recommended systems are: Red Hat Enterprise Linux 7, 8, 9; CentOS 7, 8; Amazon Linux 2; Ubuntu 16.04, 18.04, 20.04
WARNING: The current system does not match with the list of recommended systems. The installation may not work properly.
```

I've just had a look at it, the system requirements are pretty modest for the fact that it looks so modern ^^

even with an old Ubuntu LXC I can't get it to run correctly. Their “install script” has 4832 lines, of which I honestly have no idea what they do. I also absolutely don't want to install an external script, because otherwise anyone could just use an old Ubuntu VM and install the script themselves.

At the moment I would throw it out of the pipeline, and in half a year I would look again to see if anything has improved.

**Comment by jproz** [comment]
Just want to tag this for review again. 
Would be really cool addition. 
[Install](https://documentation.wazuh.com/current/installation-guide/index.html)
[Video](https://youtu.be/dwr-4CXtOso)

**Comment by tremor021** [comment]
Btw, @jproz @Deepshah0308 whats the issue with installing it via official script?

Just spin up a debian lxc, download the script from:
```
curl -sO https://packages.wazuh.com/4.10/wazuh-install.sh && sudo bash ./wazuh-install.sh -a
```
Error message about non-supported distro can be ignored, installation goes just fine:
![image](https://github.com/user-attachments/assets/1eedcaf9-b1ca-4bcd-98ed-81dc2edb5c18)
Also, it can be easily bypassed by editing dist_detect() function inside the install script, since it works just fine on debian
